Formula/p/postgrest.rb

```diff
@@ -2,8 +2,8 @@ class Postgrest < Formula
   desc "Serves a fully RESTful API from any existing PostgreSQL database"
   homepage "https://github.com/PostgREST/postgrest"
   # TODO: Try to switch `ghc@9.2` to `ghc` when postgrest.cabal allows base>=4.17
```

Comment:
still not supported yet, https://github.com/PostgREST/postgrest/blob/v12.0.1/postgrest.cabal#L78

Reply:
- https://github.com/PostgREST/postgrest/issues/3107